app/scenes/thread/thread_container.js

```diff
@@ -20,9 +21,15 @@ function makeMapStateToProps() {
     return function mapStateToProps(state, ownProps) {
         const posts = getPostsForThread(state, ownProps);
 
+        let teamId = state.entities.channels.channels[ownProps.channelId].team_id;
+        if (!teamId) {
+            // We can't make a post without a team id, so get it from the current team
+            teamId = getCurrentTeamId(state);
```

Comment:
I don't think we'll be there any time soon, but I think it's possible that we could end up with views from multiple teams at once, so I usually try to avoid using the current team wherever possible. It's the same reason that client.js doesn't have a `teamId` field while the old one in the platform app does